internal/pkg/vault/secrets_test.go

```diff
@@ -865,3 +898,170 @@ func (caller *InMemoryMockCaller) Do(req *http.Request) (*http.Response, error)
 		return nil, errors.New("unsupported HTTP method")
 	}
 }
+
+func TestHttpSecretStoreManager_GetKeys(t *testing.T) {
+	TestConnError := pkg.NewErrSecretStore("testing conn error")
+	TestConnErrorPathNotFound := pkg.NewErrPathNotFound("testing path error")
+	testData := listTestSecretsKeysData()
+	tests := []struct {
+		name              string
+		path              string
+		expectedValues    []string
+		expectError       bool
```

Comment:
This field is redundant to `expectedErrorType` . Can just check `expectedErrorType == nil` for expected error

Reply:
This was not resolved in this test like it was in other test. `expectError       bool` still present.